package.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "your-module-name",
+	"name": "mrmoco-mhc",
 	"version": "0.1.0",
```

Comment:
If this is going to be the first release, please change version here to 1.0.0 and leave in manifest.json at 0.0.0